packages/expo-modules-autolinking/scripts/ios/React-Core.modulemap

```diff
@@ -0,0 +1,6 @@
+module React {
+  umbrella "../../Public/React-Core/React"
```

Comment:
Also here 😞 Shouldn't we version it? I guess maybe we don't have to, but we need to verify that.

Reply:
yep, this is also necessary. thanks!